soc/silabs/silabs_siwx91x/Kconfig

```diff
@@ -14,3 +14,19 @@ config SOC_SILABS_SLEEPTIMER
 	bool
 	help
 	  The Sleeptimer HAL module is used for SIWX91X.
+
+config SIWX91X_SILABS_COMMANDER_SIGN
+	bool "Silicon Labs Commander Signing Support"
+	depends on BUILD_OUTPUT_BIN
+	help
+	  Activates signing of Zephyr binary with Silicon Labs Commander during build process.
+
+config SIWX91X_OTA_KEY
+	string "OTA Key for Encryption (sourced from prj.conf)"
+	help
+	  Specifies OTA key (hex) for signing/encryption with Silicon Labs Commander, from prj.conf.
+
+config SIWX91X_PRIVATE_KEY
+	string "Private Key Path for Signing (sourced from prj.conf)"
+	help
+	  Path to private key file for signing with Silicon Labs Commander, from prj.conf, relative to board directory.
```

Comment:
These options should not be displayed to the user if `SOC_FAMILY_SILABS_SIWX91X` is not selected. You need to `if SOC_FAMILY_SILABS_SIWX91X`.